securedrop/tests/common.py

```diff
@@ -5,6 +5,8 @@
 
 import gnupg
 
+# Set environment variable so config.py uses a test environment
+os.environ['SECUREDROP_ENV'] = 'test'
```

Comment:
I think all environment variables need to be removed. Instead, the relevant arguments should be passed directly, such as to object constructors. See, e.g., the [`TestDatabase` mixin class](https://github.com/freedomofpress/fingerprint-securedrop/blob/ac8d0e5928c4b2bc21d390125c0311aad88a65ef/fpsd/tests/common.py#L6) I recently added to  help solve this problem in FPSD. The SecureDrop story will be a little more complicated, and I haven't figured it all out yet, but I plan to make a PR for this. Should maybe open an issue.